tests/libstorage/storageclass.go

```diff
@@ -348,3 +351,15 @@ func isLocalPV(pv k8sv1.PersistentVolume) bool {
 func isPVAvailable(pv k8sv1.PersistentVolume) bool {
 	return pv.Spec.ClaimRef == nil
 }
+
+func CreateBlockDataVolume(virtClient kubecli.KubevirtClient, namespace string) (*cdiv1.DataVolume, error) {
```

Comment:
TBH I don't think we need to put DV creation behind a function
(which was also the case prior to this PR)
Could just use libdv directly